wazp/utils.py

```diff
@@ -261,3 +266,107 @@ def export_selected_rows_as_yaml(
             yaml.dump(row, yamlf, sort_keys=False)
 
     return
+
+
+def get_dataframes_to_combine(
+    list_selected_videos: list,
+    slider_start_end_labels: list,
+    app_storage: dict,
+):
+    # build list of h5 files for the selected videos
+    # TODO: alternative to h5: pickle? csv?
```

Comment:
Fine with h5 for now, though we could explore other formats in the future, notably `.tsv` (more robust than csv, does not break with commas) or `parquet` (could be faster).

Reply:
We import h5 so we're probably stuck with supporting it.

What do the researchers actually use day-to-day? Would they prefer a c/tsv file for spreadsheets or .rdata or...? (Apologies if we've talked about this already.) Perhaps put this on the next "coffee with Sanna" list.

Reply:
Not sure about Sanna/zoo project specifically, but there's a mix of:
* The format output by third party software (e.g. .h5)
* The default format of the tool their using (e.g. .mat, .npy etc)
* Some preference for easily viewable formats (e.g. csv)

Relatively little R usage in neuroscience. I think users are happy with any format as long as there is an ecosystem of tools around it to support what they need to do.

Reply:
Personally I would constrain the choices between:
* h5: as Sam mentioned we anyhow have to support it, since that's how deeplabcut outputs the poses.
* csv/tsv: I'm a fan of those, because of the ease of viewing, manipulating in text editors, grep, excel, etc. But perhaps things will get unwieldy when one gets to >10^5 rows.